src/main.tsx

```diff
@@ -8,12 +8,24 @@ const WidgetApp = () => {
   const appId = urlParams.get('app_id') ?? import.meta.env.VITE_CHAT_WIDGET_APP_ID;
   const botId = urlParams.get('bot_id') ?? import.meta.env.VITE_CHAT_WIDGET_BOT_ID;
   const locale = urlParams.get('locale') ?? undefined;
+  const region = urlParams.get('region') ?? undefined;
+
+  function getHost(region?: string) {
+    if (region && region.startsWith('no')) {
```

Comment:
maybe no need to add 
region.startsWith('no')?

Reply:
I added it because I thought it would be needed to run tests with test region on the latest/PR branch.